package.json

```diff
@@ -263,7 +264,7 @@
     "@metamask/message-manager": "^7.3.0",
     "@metamask/metamask-eth-abis": "^3.0.0",
     "@metamask/name-controller": "^3.0.0",
-    "@metamask/network-controller": "^12.2.0",
+    "@metamask/network-controller": "^15.0.0",
```

Comment:
This is blocked by v13 bump here https://github.com/MetaMask/metamask-extension/pull/20652

Reply:
This was inadvertently committed on this PR, removing it. thanks for flagging it though